index.js

```diff
@@ -5,6 +5,18 @@ var stream = require('stream');
 var util = require('util');
 var minify = require('html-minifier').minify;
 
+function extensionToRegExp (extensions) {
+  if (extensions instanceof RegExp) {
+    return extensions;
+  }
+
+  if (typeof extensions === 'string') {
```

Comment:
This isn't how browserify works, so I don't think we should deviate. Browserify takes an array of extensions with the period. And although it merges the extensions, that behavior doesn't make sense here – replacing the current set makes more sense.